src/App.tsx

```diff
@@ -26,70 +31,116 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  switch (sortType) {
+    case SortType.ALPHABET:
+      visibleGoods.sort(
+        (good1, good2) => good1.localeCompare(good2),
+      );
+      break;
 
-  return visibleGoods;
-}
+    case SortType.LENGTH:
+      visibleGoods.sort(
+        (good1, good2) => good1.length - good2.length,
+      );
+      break;
```

Comment:
But you've duplicated sort method, let's move switch/case inside sort method instead